x-pack/plugins/apm/server/routes/alerts/register_transaction_duration_anomaly_alert_type.ts

```diff
@@ -147,6 +151,10 @@ export function registerTransactionDurationAnomalyAlertType({
                   },
                   ...termQuery('partition_field_value', ruleParams.serviceName),
                   ...termQuery('by_field_value', ruleParams.transactionType),
+                  ...termQuery(
```

Comment:
We also have `apmMlAnomalyQuery`, would it be useful here?

Reply:
yes, I think so, thanks!

Reply:
in the end I couldn't use it, it adds a `should` for either `model_plots` or `record` and I'm only interested on `record` to fetch rules